apertium-eng.eng.dix

```diff
@@ -44623,60 +44675,204 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <e lm="Dec.">            <i>Dec.</i><par n="e.g.__abbr"/></e>
+<e lm="dept.">           <i>dept.</i><par n="e.g.__abbr"/></e>
+<e lm="Dr.">             <i>Dr.</i><par n="e.g.__abbr"/></e>
+<e lm="Dy">              <i>Dy</i><par n="e.g.__abbr"/></e>
 <e lm="e. g.">           <i>e.<b/>g.</i><par n="e.g.__abbr"/></e>
 <e lm="e.g.">            <i>e.g.</i><par n="e.g.__abbr"/></e>
 <e lm="ed.">             <i>ed.</i><par n="e.g.__abbr"/></e>
+<e lm="ep.">             <i>ep.</i><par n="e.g.__abbr"/></e>
+<e lm="Er">              <i>Er</i><par n="e.g.__abbr"/></e>
+<e lm="Es">              <i>Es</i><par n="e.g.__abbr"/></e>
 <e lm="esp.">            <i>esp.</i><par n="e.g.__abbr"/></e>
+<e lm="est.">            <i>est.</i><par n="e.g.__abbr"/></e>
 <e lm="estab.">          <i>estab.</i><par n="e.g.__abbr"/></e>
+<e lm="eta">             <i>eta</i><par n="e.g.__abbr"/></e>
 <e lm="etc.">            <i>etc.</i><par n="e.g.__abbr"/></e>
+<e lm="Eu">              <i>Eu</i><par n="e.g.__abbr"/></e>
 <e lm="Ex.">             <i>Ex.</i><par n="e.g.__abbr"/></e>
 <e lm="ex.">             <i>ex.</i><par n="e.g.__abbr"/></e>
 <e lm="excl.">           <i>excl.</i><par n="e.g.__abbr"/></e>
+<e lm="F">               <i>F</i><par n="e.g.__abbr"/></e>
 <e lm="f.">              <i>f.</i><par n="e.g.__abbr"/></e>
+<e lm="Fe">              <i>Fe</i><par n="e.g.__abbr"/></e>
 <e lm="Feb.">            <i>Feb.</i><par n="e.g.__abbr"/></e>
 <e lm="fem.">            <i>fem.</i><par n="e.g.__abbr"/></e>
+<e lm="Fm">              <i>Fm</i><par n="e.g.__abbr"/></e>
+<e lm="Fr">              <i>Fr</i><par n="e.g.__abbr"/></e>
+<e lm="ft">              <i>ft</i><par n="e.g.__abbr"/></e>
+<e lm="g">               <i>g</i><par n="e.g.__abbr"/></e>
+<e lm="Ga">              <i>Ga</i><par n="e.g.__abbr"/></e>
+<e lm="gal">             <i>gal</i><par n="e.g.__abbr"/></e>
+<e lm="Gd">              <i>Gd</i><par n="e.g.__abbr"/></e>
+<e lm="Ge">              <i>Ge</i><par n="e.g.__abbr"/></e>
+<e lm="H">               <i>H</i><par n="e.g.__abbr"/></e>
+<e lm="He">              <i>He</i><par n="e.g.__abbr"/></e>
+<e lm="Hf">              <i>Hf</i><par n="e.g.__abbr"/></e>
+<e lm="Hg">              <i>Hg</i><par n="e.g.__abbr"/></e>
+<e lm="Ho">              <i>Ho</i><par n="e.g.__abbr"/></e>
 <e lm="hp">              <i>hp</i><par n="e.g.__abbr"/></e>
+<e lm="Hs">              <i>Hs</i><par n="e.g.__abbr"/></e>
+<e lm="I">               <i>I</i><par n="e.g.__abbr"/></e>
 <e lm="i.e.">            <i>i.e.</i><par n="e.g.__abbr"/></e>
+<e lm="In">              <i>In</i><par n="e.g.__abbr"/></e>
 <e lm="incl.">           <i>incl.</i><par n="e.g.__abbr"/></e>
+<e lm="Ir">              <i>Ir</i><par n="e.g.__abbr"/></e>
 <e r="RL" lm="issue no."><i>issue<b/>no.</i><par n="e.g.__abbr"/></e>
 <e lm="Jan.">            <i>Jan.</i><par n="e.g.__abbr"/></e>
+<e lm="JD">              <i>JD</i><par n="e.g.__abbr"/></e>
+<e lm="Jun.">            <i>P</i><par n="e.g.__abbr"/></e>
```

Comment:
What is this meant to be?

Reply:
Juris Doctor?
June?

Reply:
No like why is the entry a single 'P' while the lemma is 'Jun.'